sql/src/main/antlr/OpenSearchSQLParser.g4

```diff
@@ -307,6 +307,11 @@ functionCall
     | aggregateFunction (orderByClause)? filterClause               #filteredAggregationFunctionCall
     | relevanceFunction                                             #relevanceFunctionCall
     | highlightFunction                                             #highlightFunctionCall
+    | functionWithCachedValue                                       #functionLikeConstantCall
```

Comment:
It'd be cleaner if `#functionLikeConstantCall` matched the rule name.

Reply:
Thank you, renamed in 6ef778de.